client/src/AllVideos.css

```diff
@@ -0,0 +1,199 @@
+* {
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+}
+
+.all-videos-wrapper {
+    width: 100%;
+    margin: 80px 0;
+    list-style: none;
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+}
+
+@media only screen and (min-width: 375px) {
+    .all-videos-wrapper {
+        width: 100%;
+        margin: 120px 0;
+        list-style: none;
```

Comment:
I don't think this is needed, it's not a list element

Reply:
Just checked, It doesn't appear to be needed as you suggested, thank you.